packaging/greenboot/microshift-running-check.sh

```diff
@@ -93,102 +16,7 @@ echo "STARTED"
-if ! wait_for "${WAIT_TIMEOUT_SECS}" microshift_health_endpoints_ok ; then
-    log_failure_cmd "health-readyz" "${OCGET_CMD} --raw=/readyz?verbose"
-    log_failure_cmd "health-livez"  "${OCGET_CMD} --raw=/livez?verbose"
-
-    echo "Error: Timed out waiting for MicroShift API health endpoints to be OK"
-    exit 1
-fi
-
-if lvmsShouldBeDeployed; then
-    PODS_NS_LIST+=(openshift-storage)
-    PODS_CT_LIST+=(2)
-fi
-declare -a csi_components=('csi-snapshot-controller' 'csi-snapshot-webhook')
-csi_pods_ct=0
-for csi_c in "${csi_components[@]}"; do
-    if csiComponentShouldBeDeployed "${csi_c}"; then
-        (( csi_pods_ct += 1 ))
-    fi
-done
-if [ ${csi_pods_ct} -gt 0 ]; then
-    PODS_NS_LIST+=(kube-system)
-    PODS_CT_LIST+=("${csi_pods_ct}")
-fi
-
-# Starting pod-specific checks
-# Log list of pods and their events on failure
-LOG_POD_EVENTS=true
-
-# Wait for any pods to enter running state
-echo "Waiting ${WAIT_TIMEOUT_SECS}s for any pods to be running"
-if ! wait_for "${WAIT_TIMEOUT_SECS}" any_pods_running ; then
-    echo "Error: Timed out waiting for any MicroShift pod to be running"
-    exit 1
-fi
-
-# Wait for MicroShift core pod images to be downloaded
-for i in "${!PODS_NS_LIST[@]}"; do
-    CHECK_PODS_NS=${PODS_NS_LIST[${i}]}
-
-    echo "Waiting ${WAIT_TIMEOUT_SECS}s for pod image(s) from the '${CHECK_PODS_NS}' namespace to be downloaded"
-    if ! wait_for "${WAIT_TIMEOUT_SECS}" namespace_images_downloaded; then
-        echo "Error: Timed out waiting for pod image(s) from the '${CHECK_PODS_NS}' namespace to be downloaded"
-        exit 1
-    fi
-done
-
-# Wait for MicroShift core pods to enter ready state
-for i in "${!PODS_NS_LIST[@]}"; do
-    CHECK_PODS_NS=${PODS_NS_LIST[${i}]}
-    CHECK_PODS_CT=${PODS_CT_LIST[${i}]}
-
-    echo "Waiting ${WAIT_TIMEOUT_SECS}s for ${CHECK_PODS_CT} pod(s) from the '${CHECK_PODS_NS}' namespace to be in 'Ready' state"
-    if ! wait_for "${WAIT_TIMEOUT_SECS}" namespace_pods_ready; then
-        echo "Error: Timed out waiting for ${CHECK_PODS_CT} pod(s) in the '${CHECK_PODS_NS}' namespace to be in 'Ready' state"
-        exit 1
-    fi
-done
-
-# Verify that MicroShift core pods are not restarting
-declare -A pid2name
-for i in "${!PODS_NS_LIST[@]}"; do
-    CHECK_PODS_NS=${PODS_NS_LIST[${i}]}
-
-    echo "Checking pod restart count in the '${CHECK_PODS_NS}' namespace"
-    namespace_pods_not_restarting "${CHECK_PODS_NS}" &
-    pid=$!
-
-    pid2name["${pid}"]="${CHECK_PODS_NS}"
-done
-
-# Wait for the restart check functions to complete, printing errors in case of a failure
-check_failed=false
-for pid in "${!pid2name[@]}"; do
-    if ! wait "${pid}" ; then
-        check_failed=true
-
-        name=${pid2name["${pid}"]}
-        echo "Error: Pods are restarting too frequently in the '${name}' namespace"
-    fi
-done
-
-# Exit with an error code if the pod restart check failed
-if ${check_failed} ; then
-    exit 1
-fi
```

Comment:
Should we not have a call to check microshift core namespaces here?
If it still does not work, let's add a TODO comment?

Reply:
`microshift healthcheck` without options checks the core namespaces so it works by default